docs/guide/cli.md

```diff
@@ -79,6 +79,7 @@ vitest related /src/index.ts /src/hello-world.js
 | `--inspect` | Enables Node.js inspector |
 | `--inspect-brk` | Enables Node.js inspector with break |
 | `-h, --help` | Display available CLI options |
+| `--extra-filter` | extra match file basename to test |
```

Comment:
Why extra? "extra" implies "more", here we are limiting it. It should be "exact"

Reply:
oh! sorry